persistence/persistence-api/src/main/java/org/cbioportal/persistence/MutationRepository.java

```diff
@@ -42,27 +42,7 @@ MutationMeta getMetaMutationsInMultipleMolecularProfiles(List<String> molecularP
     MutationMeta fetchMetaMutationsInMolecularProfile(String molecularProfileId, List<String> sampleIds,
                                                       List<Integer> entrezGeneIds);
 
-    @Cacheable(cacheResolver = "generalRepositoryCacheResolver", condition = "@cacheEnabledConfig.getEnabled()")
-    List<MutationCountByGene> getSampleCountByEntrezGeneIdsAndSampleIds(String molecularProfileId,
-                                                                        List<String> sampleIds,
-                                                                        List<Integer> entrezGeneIds);
-
-    @Cacheable(cacheResolver = "generalRepositoryCacheResolver", condition = "@cacheEnabledConfig.getEnabled()")
-    List<MutationCountByGene> getSampleCountInMultipleMolecularProfiles(List<String> molecularProfileIds,
-                                                                        List<String> sampleIds,
-                                                                        List<Integer> entrezGeneIds);
-    
-    @Cacheable(cacheResolver = "generalRepositoryCacheResolver", condition = "@cacheEnabledConfig.getEnabled()")
-    List<MutationCountByGene> getSampleCountInMultipleMolecularProfilesForFusions(List<String> molecularProfileIds,
-                                                                        List<String> sampleIds,
-                                                                        List<Integer> entrezGeneIds);
-
-    @Cacheable(cacheResolver = "generalRepositoryCacheResolver", condition = "@cacheEnabledConfig.getEnabled()")
-    List<MutationCountByGene> getPatientCountInMultipleMolecularProfiles(List<String> molecularProfileIds,
-                                                                         List<String> patientIds,
-                                                                         List<Integer> entrezGeneIds);
-
-    @Cacheable(cacheResolver = "generalRepositoryCacheResolver", condition = "@cacheEnabledConfig.getEnabled()")
+    @Cacheable(cacheNames = "generalRepositoryCacheResolver", condition = "@cacheEnabledConfig.getEnabled()")
```

Comment:
I just noticed this in the latest squash of the code. I think this was the attempt to address the previous comment I made, but this is in the wrong place. This should remain as cacheResover = "generalRepositoryCacheResolver". Up above in class AlterationRepository, the annotation there should change to match this. (Use cacheResolver = "generalRepositoryCacheResolver").  We should no longer be using explicit cacheNames.

Reply:
Ah, I missed the different name for the parameter. Ok, corrected and squashed.